.github/workflows/publish-snap.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish-snap
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-snap:
+    runs-on: ubuntu-latest
+
+    # Skip running workflow on forks
+    if: github.repository_owner == 'prql'
+
+    steps:
+      - name: 📂 Checkout code
+        uses: actions/checkout@v3
+      - name: 📦 Build Snap
+        uses: snapcore/action-build@v1
+        with:
+          path: prql-compiler
+      - name: 🆙 Publish Snap
+        uses: snapcore/action-publish@v1
+        env:
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.STORE_LOGIN }}
```

Comment:
I will register and add this secret in later today...

Reply:
I've added this to the repo. (Though until this PR is merged, it won't get access to it, so we should expect auth errors while we test.) 

Let's test that this gets as far as the auth error before we merge